src/Bicep.Core.Samples/Files/InvalidModules_LF/main.diagnostics.bicep

```diff
@@ -515,7 +515,7 @@ module wrongModuleParameterInLoop 'modulea.bicep' = [for x in emptyArray:{
     stringParamA: 'test'
     stringParamB: 'test'
     notAThing: 'test'
-//@[4:13) [BCP037 (Error)] No other properties are allowed on objects of type "params". |notAThing|
```

Comment:
>BCP037 [](http://example.com/codeflow?start=11&length=6)

We added that error message to address some issue in the past, so we shouldn't regress on this.

Reply:
Brought it back.